File: backend/models/item.go
package models

type Item struct {
	ID         int     `json:"id"`
	CategoryID int     `json:"category_id"`
	Brand      string  `json:"brand"`
	Name       string  `json:"name"`
	Duration   string  `json:"duration"`
	Price      float64 `json:"price"`
	Field      string  `json:"field"`
}

File: frontend/lib/models/item.dart
class Item {
  final int id;
  final int categoryId;
  final String brand;
  final String name;
  final String duration;
  final double price;
  final String field;

  Item({
    required this.id,
    required this.categoryId,
    required this.brand,
    required this.name,
    required this.duration,
    required this.price,
    required this.field,
  });

  factory Item.fromJson(Map<String, dynamic> json) {
    return Item(
      id: json['id'],
      categoryId: json['category_id'],
      brand: json['brand'],
      name: json['name'],
      duration: json['duration'],
      price: (json['price'] as num).toDouble(),
      field: json['field'] ?? '',
    );
  }
}
